src/main.cpp

```diff
@@ -69,6 +69,24 @@ std::string exec(const char* cmd) {
     return result;
 }
 
+void open_output_file(const std::string& a, const std::string& b)
+{
+    std::filesystem::path path(b);
+    if (!a.empty()) {
+        path = a + '.' + b;
+    }
+    std::cout << "open output file\n";
+    std::cout << "path:" << path << "\n";
```

Comment:
Are these intended to remain? Should they be `cout_safe`?

Reply:
oops - thanks!